Tests/HTTPTransactionTests.swift

```diff
@@ -57,6 +57,34 @@ class HTTPTransactionTests: XCTestCase {
         XCTAssertNotNil(configuration.authorization, "The authorization has not been initialized")
     }
 
+    func testSetup_withConfiguration_configReturned() {
```

Comment:
This test should not be here. This test class is only for HTTPTransactions and not for Hyperwallet

Reply:
Right! should be in HyperwalletTests